More Hairstyles

Munnymouse opened the pull request:
:cl:
add: New human hairstyles added
/:cl:

This contains 39(!) more hairstyles that have been ported from Aurora to allow for greater character customization. Some have been tweaked/re-shaded to work with Bay's style.

Massive thanks to everybody in coding (alex/aftershot/loaf/psi/tiviplus in particular!) for holding my hand and listening to my silly questions as I completed this for my first PR.

Munnymouse commented:
Sorry for the slightly messy PR. The commits have been me chasing down rogue hairstyles and getting used to how everything works, but it should now be good.

Spookerton commented on code/modules/sprite_accessories/_accessory_hair.dm:
```diff
@@ -267,7 +267,7 @@
 	flags = HAIR_TIEABLE
 
-/datum/sprite_accessory/hair/shortovereye
+/datum/sprite_accessory/hair/shorteye
```
repath invalidates existing setups

PsiOmegaDelta replied:
Character preferences are based on name, not path

Spookerton replied:
oop.

Spookerton commented on code/modules/sprite_accessories/_accessory_hair.dm:
```diff
@@ -729,3 +725,179 @@
+	name = "Belenkotied"
+	icon_state = "hair_belenkotied"
+
+/datum/sprite_accessory/hair/bobkusanagi
+	name = "Kusanagi Bob"
+	icon_state = "hair_bob_kusanagi"
```
duplicate of `hair_kusanagi`

Spookerton commented on code/modules/sprite_accessories/_accessory_hair.dm:
```diff
@@ -729,3 +725,179 @@
 	icon_state = "hair_rightside_trim"
 	flags = HAIR_TIEABLE
+
+/datum/sprite_accessory/hair/vshortbangs
+	name = "Very Short Bangs"
+	icon_state = "hair_bangs_veryshort"
```
duplicate of `hair_shortbangs`
re*name invalidates existing setups

PsiOmegaDelta replied:
Seems like `flags = VERY_SHORT` is applicable here

Spookerton commented on code/modules/sprite_accessories/_accessory_hair.dm:
```diff
@@ -729,3 +725,179 @@
+	name = "Short Bun"
+	icon_state = "hair_bun_short"
+
+/datum/sprite_accessory/hair/coffeehouseshave
```
duplicate of `hair_shavedpart`

Spookerton commented on code/modules/sprite_accessories/_accessory_hair.dm:
```diff
@@ -729,3 +725,179 @@
+	name = "Emo Fringe"
+	icon_state = "hair_emofringe"
+
+/datum/sprite_accessory/hair/longemofringe
```
duplicate of `hair_emolong`

PsiOmegaDelta commented:
You should go over and check whether the `HAIR_TIEABLE` and/or `HAIR_SHORT` flags are applicable in general

Spookerton commented on code/modules/sprite_accessories/_accessory_hair.dm:
```diff
@@ -729,3 +725,179 @@
+	name = "Modern"
+	icon_state = "hair_modern"
+
+/datum/sprite_accessory/hair/mohawkfront
```
duplicate of `hair_mohawkshaved`

PsiOmegaDelta commented:
Hair with the `HAIR_TIEABLE`-flag is long enough for the owner to retie it into other hairstyles with the `HAIR_TIEABLE`-flag
The `VERY_SHORT`-flag is mostly cosmetic, but will have different messages when combed through and will be completely hidden if a helmet or the like is put on.

Spookerton commented on code/modules/sprite_accessories/_accessory_hair.dm:
```diff
@@ -460,10 +460,6 @@
 	icon_state = "hair_longeralt2"
 	flags = HAIR_TIEABLE
 
-/datum/sprite_accessory/hair/shortbangs
```
should be okay after you revert this removal.

Munnymouse replied:
Whoops. Don't think I meant that! Reverted + tested.

Munnymouse committed:
More Hairstyles

This contains 50(!) more hairstyles that have been ported from Aurora to allow for greater character customization. 

Some have been tweaked/re-shaded to work with Bay's style.
More Hairstyles

This contains 48(!) more hairstyles that have been ported from Aurora to allow for greater character customization. Some have been tweaked/re-shaded to work with Bay's style.

Massive thanks to everybody in coding (alex/aftershot/loaf/psi/tiviplus in particular!) for holding my hand and listening to my silly questions as I completed this for my first PR.
More Hairstyles

More hairstyles
more hairstyles

more hairstyles
hairstyles

hairsyle
Fixes & Tags

Duplicates have been removed, and tags have been added to appropriate hairstyles.
Additional Fix

Fixing a final duplicate.
Naming changes

A minor name change to two hairstyles.
removed VERY_SHORT flag for the mohawk hairs
Revert Short Bangs

Munnymouse commented:
I think Travis needs to re-run? It should be good to go beyond that.
